### PR TITLE
Order Placement

### PR DESCRIPTION
**Pull request by anabaarbosa**
adding the Order Placement response schema

#### Types of changes
- [x] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

### CODE REVIEW

File: VTEX - Marketplace Protocol - External Seller Fulfillment.json [hunk 1]
```diff
@@ -1149,40 +1149,43 @@
...
-                                "examples": {
+                                "schema":{
+                                    "$ref": "#/components/schemas/responseOrderPlacement"
+                                },
+                                "example": {
                                     "response": {
                                         "value": [
                                             {
                                                 "marketplaceOrderId": "959311095",
-                                                "marketplaceServicesEndpoint": "https://marketplaceservicesendpoint/",
```
Comment (jumeyermattosarruda) [inline VTEX - Marketplace Protocol - External Seller Fulfillment.json:1157]: Esses dois campos foram removidos mesmo?
- marketplaceServicesEndpoint
- marketplacePaymentValue

File: VTEX - Marketplace Protocol - External Seller Fulfillment.json [hunk 2]
```diff
@@ -1149,40 +1149,43 @@
...
                                                 ],
                                                 "clientProfileData": {
-                                                    "email": "32172239852@gmail.com.br",
+                                                    "email": "customer@examplemail.com.br",
                                                     "firstName": "John",
                                                     "lastName": "Smith",
-                                                    "documentType": null,
-                                                    "document": "3244239851",
-                                                    "phone": "399271258",
+                                                    "documentType": "cpf",
```
Comment (jumeyermattosarruda) [inline VTEX - Marketplace Protocol - External Seller Fulfillment.json:1186]: Pra deixar o conteúdo mais globalizado, ao invés de CPF. Abri uma thread com essa [discussão](https://vtex.slack.com/archives/C052CEU8GDU/p1681998119917139)

**Review by jumeyermattosarruda — APPROVED** [review]

File: VTEX - Marketplace Protocol - External Seller Fulfillment.json (hunk 1 above)
Comment (anabaarbosa) [reply VTEX - Marketplace Protocol - External Seller Fulfillment.json:1157]: Ju, confirmei com o Rodolfo que a response e o exemplo estão corretos, esses campos não estão mais na response aguardada

File: VTEX - Marketplace Protocol - External Seller Fulfillment.json [hunk 3]
```diff
@@ -1880,6 +1885,366 @@
...
+                                    }
+                                },
+                                "attachments": {
+                                    "type": "array",
+                                    "description": "Array containing information on attachments.",
+                                    "items": {}
+                                },
+                                "priceTags": {
+                                    "type": "array",
+                                    "description": "Array of price tags, each of which, modifies the price in some way, like discounts or rates that apply to the item in the context of the order.",
```
Comment (jumeyermattosarruda) [inline VTEX - Marketplace Protocol - External Seller Fulfillment.json:1960]:
```suggestion
                                    "description": "Array of price tags. Each one modifies the price in some way, like discounts or rates that apply to the item in the context of the order.",
```

**Review by jumeyermattosarruda — DISMISSED** [review]

**Pull request merged**